### ISSUE TITLE
y_groups warning (redefinition of constant/macro)

### ISSUE THREAD

**Issue by ghost** [pawn-lang/YSI-Includes#184]
I get a warning when I included y_groups, it seems that has duplicate an iterator with the same name.

```
// This is a comment
// uncomment the line below if you want to write a filterscript
//#define FILTERSCRIPT

#include <a_samp> //SA-MP Team

#include <YSI_Players\y_groups>

main()
{
	print("\n----------------------------------");
	print(" Blank Gamemode by your name here");
	print("----------------------------------\n");
}
```
```
C:\SAMP Server\pawno\include\YSI_Players\y_groups/impl.inc(1239) : warning 201: redefinition of constant/macro (symbol "Iterator@Group")
Pawn compiler 3.10.2	 	 	Copyright (c) 1997-2006, ITB CompuPhase


1 Warning.
```

I have a very stupid suggestion because maybe I am the only one who needs it. 

I use strlib of Slice, 'sprintf' is in strlib and y_utils include, I get an error, it would be nice if you use in functions that may be in other include

```
#if defined sprintf
#else
#endif
```

fix please, thanks

**Comment by Y-Less** [comment]
Thanks.  I'll investigate this.  Frankly I'm not sure how I missed that, though y_groups hasn't been in my testing rounds for a while as I've not been working on it.

**Comment by Y-Less** [comment]
Fixed via 25411f4

**Comment by ghost** [comment]
Thanks @Y-Less !
